.github/ISSUE_TEMPLATE/feature_report.yml

```diff
@@ -0,0 +1,55 @@
+name: 💡 General Feature Request
+description: Have a new idea/feature for rbjks? Let us know...
+title: "[FEATURE] <description>"
+labels: ["⭐ goal: addition", "🚦 status: awaiting triage"]
```

Comment:
@adityaraute I need your opinion on the labels here, they look fine to me but up to you if you want emojis or a different label text

Reply:
@adityaraute @agriyakhetarpal resolved all the changes wait for emoji response for pull request

Reply:
@Latish705 I don't think we need these labels here. Can you instead replace these with a simple label called `enhancement`

We can maybe think more on this later, but for now let's keep it simple.

Reply:
@adityaraute done